observations/obs_converters/AIRS/airs_JPL_mod.f90

```diff
@@ -150,28 +150,23 @@ subroutine airs_ret_rdr(file_name, airs_ret_gran, ver)
                                 ! 1 => use every element
       integer :: edge(10)       ! size of each dimension for swath I/O
                                 ! will be set for each individual read
-
-      ! Using the HDF-EOS5 functions simply required
-      ! putting an 'he5_' in front of the function. As far as I can tell from
-      ! https://hdfeos.org/examples/fort_he5_swath.php, the calling structure is
-      ! the same.
       
-      integer :: he5_swopen         ! open a swath file
-      integer :: he5_swattach       ! attatch to a swath object
-      integer :: he5_swinqswath     ! retieves number and names of swaths in file
-      integer :: he5_swrdfld        ! read data from a data field
-      integer :: he5_swrdattr       ! read swath attribute
-      integer :: he5_swdetach       ! detatching from the swath object
-      integer :: he5_swclose        ! closing the file
-
-      fid = he5_swopen(file_name, 1)
+      integer :: swopen         ! open a swath file
+      integer :: swattach       ! attatch to a swath object
+      integer :: swinqswath     ! retieves number and names of swaths in file
+      integer :: swrdfld        ! read data from a data field
+      integer :: swrdattr       ! read swath attribute
+      integer :: swdetach       ! detatching from the swath object
+      integer :: swclose        ! closing the file
```

Comment:
At one point, this module used HDF-EOS  calls and I converted them to use HDF-EOS5 calls when I thought it had to use the RTTOV libraries.  Turns out the converter that uses this module (`convert_airs_L2.f90`) does not use radiances, so I am
reverting back to the aa9e25892ffc  version.